pom.xml

```
@@ -25,6 +27,18 @@
             <version>4.6.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5</artifactId>
+            <version>2.10.2.Final</version>
+            <scope>test</scope>
+        </dependency>
```

```suggestion
         <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter</artifactId>
            <version>5.9.0-M1</version>
            <scope>test</scope>
        </dependency>
```
У тебя не та библиотека